sdk/search/Azure.Search.Documents/README.md

```diff
@@ -214,14 +214,20 @@ Let's explore them with a search for a "luxury" hotel.
 We can decorate our own C# types with [attributes from `System.Text.Json`](https://docs.microsoft.com/dotnet/standard/serialization/system-text-json-how-to):
 
 ```C# Snippet:Azure_Search_Tests_Samples_Readme_StaticType
-public class Hotel
-{
-    [JsonPropertyName("hotelId")]
-    public string Id { get; set; }
-
-    [JsonPropertyName("hotelName")]
-    public string Name { get; set; }
-}
+        public class Hotel
+        {
+            [JsonPropertyName("hotelId")]
+#if EXPERIMENTAL_FIELDBUILDER
```

Comment:
Do we need the flag at this point?

Reply:
@tg-msft and @AlexGhiondea wanted it. We'll almost definitely end up shipping it and should remove it later.